assets/source/components/prelaunch-notice/index.js

```diff
@@ -25,9 +36,15 @@ const PrelaunchNotice = () => {
 			</p>
 			<p>
 				<a
-					href="https://help.pinterest.com/en-gb/business/article/get-a-business-profile"
-					target="_blank"
-					rel="noreferrer"
+					{ ...documentationLinkProps( {
+						href:
+							wcSettings.pinterest_for_woocommerce.pinterestLinks
+								.preLaunchNotice,
+						eventName: 'get_started_notice_link_click',
+						linkId: 'prelaunch-notice',
+						context: 'pinterest-landing', //todo: Use helper function to get context from path see https://github.com/woocommerce/pinterest-for-woocommerce/pull/295
```

Comment:
I think we didn't settle as a team on generating  `context`, so I'd rather put this comment as an issue or discuss it in P2 than keep a todo in code.

Reply:
Forgot about this one. Thanks.

Fixed in https://github.com/woocommerce/pinterest-for-woocommerce/pull/299/commits/e1a61818704cd3d2f7d23078aa83b414d3e2b22e